docs/DeveloperGuide.md

```diff
@@ -602,29 +602,28 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 | Priority | As a … | I want to … | So that I can…|
 | - | - | - | - |
-| `* * *` | new user | see usage instructions | refer to instructions if I forget how to use the app |
 | `* * *` | confused user | have a help command | learn how to use the application |
-| `* * *` | user | add a new person | |
-| `* * *` | user | add issues | |
-| `* * *` | user | see a list of pending tasks sorted by deadline | prioritise which to do |
-| `* * *` | user | search for tasks | |
+| `* * *` | user | add issues | track associated issues with rooms and their status|
+| `* * *` | user | see a list of pending issues sorted by deadline | prioritise which issue to work on |
+| `* * *` | user | search for issues | find a specific issue and get its information |
+| `* * *` | user | search for issues | find issues matching a specific criteria |
 | `* * *` | user | close issues | focus on open issues |
-| `* * *` | user | update issues | |
-| `* * *` | user | remove issues | |
-| `* * *` | user | add residents | |
-| `* * *` | user | search for residents | |
+| `* * *` | user | update issues | ensure it has the latest and correct information |
+| `* * *` | user | remove issues | not worry about issues that are no longer relevant |
+| `* * *` | user | add residents | track and manage residents who are staying or want to stay in the hostel |
+| `* * *` | user | search for residents | find a specific resident's information |
 | `* * *` | user | update resident details | ensure their information is up to date |
-| `* * *` | user | remove residents | |
-| `* * *` | user | add new rooms | |
+| `* * *` | user | remove residents | stop tracking residents who are not in the hostel or don't wish to stay anymore |
+| `* * *` | user | add new rooms | track and manage rooms in the hostel |
 | `* * *` | user | see a list of rooms with their statuses | know which rooms are available |
-| `* * *` | user | search for rooms | |
-| `* * *` | user | update room details | |
-| `* * *` | user | remove rooms | |
+| `* * *` | user | search for rooms | find a specific room or rooms matching a specific criteria |
+| `* * *` | user | update room details | ensure a room's information is up to date |
+| `* * *` | user | remove rooms | stop tracking rooms which are not in the hostel |
 | `* *` | user | see both available rooms and unassigned residents side by side | easily match residents to rooms |
```

Comment:
Is this user story still relevant?

> see both available rooms and unassigned residents side by side

Reply:
It isnt as we cannot filter by unoccupied rooms. I will remove it.